src/test/java/io/github/externschool/planner/service/UserServiceTest.java

```diff
@@ -60,12 +64,16 @@ public void shouldReturnUserWhenCreateNewUser(){
                 .isEqualTo(userDTO.getEmail());
         assertThat(actualUser.getPassword())
                 .isEqualTo(userDTO.getPassword());
+        assertTrue(actualUser.getRoles()
+                .contains(new Role("GUEST")));
     }
 
     @Test
-    public void shoudReturnUserWhenFindUserByEmail(){
+    public void shouldReturnUserWhenFindUserByEmail(){
         User actualUser = userService.findUserByEmail(expectedUser.getEmail());
 
+        verify(userRepository).findByEmail(actualUser.getEmail());
```

Comment:
This verification is anti-pattern.

You should test behavior, but you  tested concrete implementation.

If you change internal logic for testable method and replace repository to another method saving data then this test is fail.
This is bad, because we should encapsulate internal logic for test

see more https://habr.com/post/358178/#5